[MISC] Import Heracles Form Changes

acrowthe opened the pull request:
Import latests Heracles form changes from spreadsheet.
6MWT: Add unit to age, fix crutches typo, add symptom severity question (was a 6mwd test performed = true > were any symptoms experienced = true > Rate the severity)
CPET External: Fix duplicated question ID that was causing a missing question
CPET Interpretation: Added new questions (Currently prescribed <beta/calcium> channel blockers), added severity question, add more options to protocol
Event Clinical Cardiac: Made event class required, fixed conditions for `stress test results` question
Event Non-Cardiac: Added other field to diagnosis question
Laboratory Results: Added CRP question
Historical Laboratory Results: Added CRP and pro-BNP questions

acrowthe committed:
[MISC] Import Heracles Form Changes
Import latests Heracles form changes from spreadsheet.
6MWT: Add unit to age, fix crutches typo
CPET External: Fix duplicated question ID that was causing a missing question
CPET Interpretation: Added new questions (Currently perscribed <beta/calcium> channel bloackers)
Event Clinical Cardiac: Made event class required
Laboratory & Historical Laboratory Results: Added pro BNP and CRP questions

sdumitriu commented on heracles-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/CPET Interpretation.xml:
```diff
@@ -1763,11 +1763,11 @@
 			<property>
 				<name>text</name>
-				<value>Did the patient take a beta-blocker today?</value>
+				<value>Is the participant curently prescribed beta-blockers?</value>
```
```suggestion
				<value>Is the participant currently prescribed beta-blockers?</value>
```

sdumitriu commented on heracles-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/CPET Interpretation.xml:
```diff
@@ -1791,59 +1791,97 @@
+			<primaryNodeType>cards:Question</primaryNodeType>
+			<property>
+				<name>text</name>
+				<value>Is the participant curently prescribed calcium-channel blockers?</value>
```
```suggestion
				<value>Is the participant currently prescribed calcium-channel blockers?</value>
```

sdumitriu committed:
[misc] Fixed typos

acrowthe committed:
[MISC] Import Heracles Form Changes
Manually fix Event - Non-Cardiac and Event - Clinical Cardiac import bugs

acrowthe committed:
[MISC] Import Heracles Form Changes
Fix conditional import bug

acrowthe committed:
[MISC] Import Heracles Form Changes
Import requested changes
- Add symptom severity questions to 6MWT and CPET Interpretation
- Remove pro-BNP from Laboratory Results

acrowthe committed:
[MISC] Import Heracles Form Changes
Add more options for CPET interpretation protocol
